external/json/cJSON.c

```diff
@@ -416,10 +389,8 @@ static unsigned char* ensure(printbuffer * const p, size_t needed)
     }
     p->length = newsize;
     p->buffer = newbuffer;
-
     return newbuffer + p->offset;
 }
-
```

Comment:
As per the coding guidelines One line spacing between function is recommended for readability

May i know why it has been removed ?